codegen/sdk-codegen/aws-models/s3.2006-03-01.json

```diff
@@ -4233,7 +4233,8 @@
         "Policy": {
           "target": "com.amazonaws.s3#Policy",
           "traits": {
-            "smithy.api#documentation": "<p>The bucket policy as a JSON document.</p>"
+            "smithy.api#documentation": "<p>The bucket policy as a JSON document.</p>",
+            "smithy.api#httpPayload": {}
```

Comment:
Why is this manually updated?

Reply:
It was manually updated, as it's missing in the copy of models in codegen.
I verified that aws-models has httpPayload, so it's won't get reverted when models get updated.